docs/quickstart.rst

```diff
@@ -1,4 +1,83 @@
 .. _quickstart:
 
 Quickstart
-==========
\ No newline at end of file
+==========
+
+Installation
+------------
+
+.. raw:: html
+
+    <div class="container">
```

Comment:
I'm not too crazy about directly doing HTML here. Can we keep it simple and stick to .rst for the time being? Once we have the landing page with its CSS files, we can have a more holistic look at fancy styling, but for now, I'd rather just stick to sections / lists / code quotes etc.

Reply:
Sure